.github/workflows/ci.yml

```diff
@@ -94,9 +94,6 @@ jobs:
       matrix:
         include:
         ### Analysis builds
-        - name: linux_gcc-10_debug_analysis_omp5
-          os: ubuntu-latest
-          env: {CXX: g++,     CC: gcc,    ALPAKA_CI_GCC_VER: 10,       ALPAKA_CI_STDLIB: libstdc++, CMAKE_BUILD_TYPE: Debug,   ALPAKA_CI_BOOST_BRANCH: boost-1.74.0, ALPAKA_CI_CMAKE_VER: 3.18.6, OMP_NUM_THREADS: 4, ALPAKA_CI_DOCKER_BASE_IMAGE_NAME: "ubuntu:20.04", ALPAKA_CI_ANALYSIS: ON, ALPAKA_DEBUG: 2, ALPAKA_ACC_ANY_BT_OMP5_ENABLE: ON, CMAKE_CXX_FLAGS: "-foffload=disable"}
```

Comment:
It only was an analysis build because running the tests did not work. It was rather changed to be a full test.

Reply:
The analysis build runs checks (like the header-check) that are not done in non-analysis builds. So by removing this one, you loose those checks on the OpenACC related headers. I would rather keep the test, but I have no strong opinion.

Reply:
This is new in a4ab9eee9e5 (last month), I was not aware of this change. Originally, the header check was also performed in all the other tests and the analysis build did not run the tests.

My intent, and the original intent of the existing `_omp5` and `_oacc` tests is to doe the header check and run the tests. Only the GCC 10 test originally was supposed to only compiler but not run, because running did not work.

Thanks you pointing this out. I will set `ALPAKA_CHECK_HEADERS=ON` then for all tests.

Reply:
The header check was there for a long time, we just restricted it to the analysis builds to speedup the CI. Furthermore, the header check is no longer enabled by default, so when you build locally, it is not run.

Reply:
> The header check was there for a long time

Indeed. It just did not notice, it was not build by-default anymore. I have to adjust to this.

Reply:
I added the header check for the `_oacc` and `_omp5` jobs (eafe84f409). There is only one per (compiler,target), so having a separate analysis job for each would add more setup time than it would save in sequential build time.

Reply:
Gitlab does not like what I did to the [`gitlab-ci.yml`](https://github.com/alpaka-group/alpaka/pull/1564/commits/eafe84f4099e29e04e5e1b9074947a65a725a560#diff-72d96b76899af358610624393ce5344832f5d04aa6bbd5481871029b91faedec) in eafe84f . I do not have access to to the ci-lint there. Can someone with access please tell me what is wrong? Are end-of-line comments not allowed? Or does the `ON` need to be quoted?